cms/core/blocks.py

```diff
@@ -66,6 +95,8 @@ class CoreBlocks(StreamBlock):
     summary_list = SummaryListBlock(group="Base")
     promo = CardImageBlock(group="Base")
     promo_group = CardGroupBlock(group="Base")
+    page_heading = PageHeadingBlock(group="Custom")
```

Comment:
It'd be nice to separate out the CoreBlocks that are a part of the NHS UK Frontend from the other blocks -- I'd add a comment here to give some visual distinction.

Reply:
The plan is to rearrange all of the blocks into more logical sections (eg 'self-updating', 'promo', 'list', 'content flow') within the admin, so it would make sense to do this as part of that.